### PR TITLE
Correcting quay login command and updating example as per GCP Artifac…

### PR DESCRIPTION
**Pull request by rajeev-asthana**
…t Registry

Correcting quay login command and updating example as per GCP Artifact Registry as GCR is deprecated now.

### CODE REVIEW

File: docs/content/preview/yugabyte-platform/install-yugabyte-platform/prepare-environment/kubernetes.md [hunk 1]
```diff
@@ -250,30 +250,38 @@ You need to perform the following steps:
   docker pull nginxinc/nginx-unprivileged:1.23.3
   ```
 
-- Login to your target container registry, as per the following example that uses Google Container Registry (GCR):
+- Login to your target container registry, as per the following example that uses Google Artifact Registry:
+
+   **Note**: The "Service Account" and "Location" used in the below example have to be replaced as applicable.
```
Comment (ddhodge) [inline docs/content/preview/yugabyte-platform/install-yugabyte-platform/prepare-environment/kubernetes.md:255]:
```suggestion
   **Note**: Replace the Service Account and Location in the example as applicable.
```

File: docs/content/preview/yugabyte-platform/install-yugabyte-platform/prepare-environment/kubernetes.md [hunk 2]
```diff
@@ -250,30 +250,38 @@ You need to perform the following steps:
...
 
   ```sh
-  docker login -u _json_key --password-stdin https://gcr.io < .ssh/my-service-account-key.json
+  gcloud auth activate-service-account yugabytedb-test@yugabytedb-test-384308.iam.gserviceaccount.com --key-file=key.json
+  gcloud auth configure-docker us-central1-docker.pkg.dev
+  
   ```
 
-- Tag the local images to your target registry, as follows:
+- Tag the local images to your target registry, as follows (Here format for GCP Artifact Registry ):
```
Comment (ddhodge) [inline docs/content/preview/yugabyte-platform/install-yugabyte-platform/prepare-environment/kubernetes.md:263]:
```suggestion
- Tag the local images to your target registry. The following example uses Google Artifact Registry.
```

File: docs/content/preview/yugabyte-platform/install-yugabyte-platform/prepare-environment/kubernetes.md [hunk 3]
```diff
@@ -250,30 +250,38 @@ You need to perform the following steps:
...
+
+  **Note**: The Location, Project Id, Repository, and Image used in the below example have to be replaced as applicable.
```
Comment (ddhodge) [inline docs/content/preview/yugabyte-platform/install-yugabyte-platform/prepare-environment/kubernetes.md:265]:
```suggestion
  **Note**: Replace the Location, Project ID, Repository, and Image in the example as applicable.
```

File: docs/content/preview/yugabyte-platform/install-yugabyte-platform/prepare-environment/kubernetes.md [hunk 4]
```diff
@@ -250,30 +250,38 @@ You need to perform the following steps:
...
+  docker tag quay.io/yugabyte/yugaware:{{<yb-version version="preview" format="build">}} us-central1-docker.pkg.dev/yugabytedb-test-384308/yugabytepoc/yugabyte/yugaware:{{<yb-version version="preview" format="build">}}
+  docker tag nginxinc/nginx-unprivileged:1.23.3 us-central1-docker.pkg.dev/yugabytedb-test-384308/yugabytepoc/nginxinc/nginx-unprivileged:1.23.3
+  docker tag postgres:14.4 us-central1-docker.pkg.dev/yugabytedb-test-384308/yugabytepoc/postgres:14.4
+  docker tag tianon/postgres-upgrade:11-to-14 us-central1-docker.pkg.dev/yugabytedb-test-384308/yugabytepoc/tianon/postgres-upgrade:11-to-14
+  docker tag prom/prometheus:v2.41.0 us-central1-docker.pkg.dev/yugabytedb-test-384308/yugabytepoc/prom/prometheus:v2.41.0
   ```
 
 - Push images to the private container registry, as follows:
 
+  **Note**: The Location, Project Id, Repository, and Image used in the below example have to be replaced as applicable.
```
Comment (ddhodge) [inline docs/content/preview/yugabyte-platform/install-yugabyte-platform/prepare-environment/kubernetes.md:277]:
```suggestion
  **Note**: Replace the Location, Project ID, Repository, and Image in the example as applicable.
```

**Review by ddhodge — CHANGES_REQUESTED** [review]

File: docs/content/preview/yugabyte-platform/install-yugabyte-platform/prepare-environment/kubernetes.md [hunk 5]
```diff
@@ -250,30 +250,38 @@ You need to perform the following steps:
...
+
+  **Note**: The Location, Project Id, Repository, and Image used in the below example have to be replaced as applicable.
 
   ```sh
-  docker tag quay.io/yugabyte/yugaware:{{<yb-version version="preview" format="build">}} gcr.io/dataengineeringdemos/yugabyte/yugaware:{{<yb-version version="preview" format="build">}}
-  docker tag nginxinc/nginx-unprivileged:1.23.3 gcr.io/dataengineeringdemos/nginxinc/nginx-unprivileged:1.23.3
-  docker tag postgres:14.4 gcr.io/dataengineeringdemos/postgres:14.4
-  docker tag tianon/postgres-upgrade:11-to-14 gcr.io/dataengineeringdemos/tianon/postgres-upgrade:11-to-14
-  docker tag prom/prometheus:v2.41.0 gcr.io/dataengineeringdemos/prom/prometheus:v2.41.0
+  docker tag quay.io/yugabyte/yugaware:{{<yb-version version="preview" format="build">}} us-central1-docker.pkg.dev/yugabytedb-test-384308/yugabytepoc/yugabyte/yugaware:{{<yb-version version="preview" format="build">}}
```
Comment (iSignal) [inline docs/content/preview/yugabyte-platform/install-yugabyte-platform/prepare-environment/kubernetes.md:268]: @rajeev-asthana : can you clarify why we want to change these links from gcr.io to us-central1.docker.pkg.dev?

Comment (rajeev-asthana) [reply docs/content/preview/yugabyte-platform/install-yugabyte-platform/prepare-environment/kubernetes.md:268]: @iSignal :   GCR  i.e. Google container registry is deprecated now and is replaced by Google Artifact Registry. Example here is using GCR which needs to be updated with Artifact registry example. "us-central1.docker.pkg.dev" is the syntax for pushing images to Artifact registry in GCP.

**Review by bhavin192 — APPROVED** [review]
Thank you for submitting the PR. 
The main goal of this document is to keep the instructions general rather than specific to one registry, seems like with pkg.dev the login instructions have become specific to gcloud. 

I think we will merge this now and figure out generic instructions which will apply to any registry. Those who are using registries with specific requirement for authentication, would already know how to do that.

**Review by ddhodge — APPROVED** [review]

**Pull request merged**